libs/common/src/vault/models/data/fido2-credential.data.ts

```diff
@@ -0,0 +1,35 @@
+import { Fido2CredentialApi } from "../../api/fido2-credential.api";
+
+export class Fido2CredentialData {
+  credentialId: string;
+  keyType: "public-key";
```

Comment:
**nit-pick (non-blocking)**:  Magic strings can be constants or custom string literal types.

Reply:
True, this could've been converted into a type alias and put somewhere else

```suggestion
  keyType: KeyType;
```